nextcord/message.py

```diff
@@ -671,7 +672,7 @@ def __init__(
         self.flags: MessageFlags = MessageFlags._from_value(data.get('flags', 0))
         self.mention_everyone: bool = data['mention_everyone']
         self.tts: bool = data['tts']
-        self.content: str = data['content']
+        self.content: Optional[str] = data['content']
```

Comment:
This could error if data has no key content... I would propose using data.get instead if it is now optional

Reply:
thanks

Reply:
resolved